WooCommerce/Classes/POS/Utils/PreviewHelpers.swift

```diff
@@ -1,6 +1,7 @@
 import Foundation
 import protocol Yosemite.POSItemProvider
 import protocol Yosemite.POSItem
+import enum Networking.ProductType
```

Comment:
nit: in the app layer, we want to avoid importing from Networking/Storage layers directly instead of Yosemite.

```suggestion
import enum Yosemite.ProductType
```